### PR TITLE
Implement Identity UI

### PR DESCRIPTION
**Pull request by AlexisTabin**
Very simple UI to show a QR code

### CODE REVIEW

File: app/src/main/java/com/github/dedis/student20_pop/OrganizerActivity.java [hunk 1]
```diff
@@ -55,6 +63,22 @@ public void onClick(View view) {
...
+                bundle.putString(PRIVATE_KEY_TAG, app.getPerson().getAuthentication());
+
+                //TODO : Retrieve this LAO from the Intent
+                Lao lao = new Lao("LAO I just joined", new Date(), new Keys().getPublicKey());
+                bundle.putString(LAO_ID_TAG, lao.getId());
+
+                // set Fragmentclass Arguments
+                IdentityFragment identityFragment = new IdentityFragment();
+                identityFragment.setArguments(bundle);
+                showFragment(identityFragment, IdentityFragment.TAG);
```
Comment (jbmichel14) [inline app/src/main/java/com/github/dedis/student20_pop/OrganizerActivity.java:79]: Is the identity UI the same if we're attendee or organizer ?

File: app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    }
+                });
+
+        //TODO : get User personal key
+        //NOTE : I was not sure which unique id to use to generate the QR code
+        //User identity is composed of :
+        //User's public key
+        //Organization's ID
+        String key = this.getArguments().getString(OrganizerActivity.PRIVATE_KEY_TAG);
+        String lao = this.getArguments().getString(OrganizerActivity.LAO_ID_TAG);;
```
Comment (jbmichel14) [inline app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java:76]: There are 2 semicolons at the end of the line

File: app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+        // TODO :   The QR code does not appear at all unless the Name field is non-empty
+        //  and not all whitespace.
+        CheckBox anonymousCheckBox = view.findViewById(R.id.checkbox_anonymous);
+        qrCode = view.findViewById(R.id.qr_code);
+        identityEmailEditText = view.findViewById(R.id.identity_email);
+        identityNameEditText = view.findViewById(R.id.identity_name);
+        identityOrganizationEditText = view.findViewById(R.id.identity_organization);
+        identityPhoneEditText = view.findViewById(R.id.identity_phone);
+        identityTitleEditText = view.findViewById(R.id.identity_title);
```
Comment (jbmichel14) [inline app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java:52]: Who (other than the user) can see these when the user is not anonymous?

File: app/src/main/java/com/github/dedis/student20_pop/OrganizerActivity.java (hunk 1 above)
Comment (AlexisTabin) [reply app/src/main/java/com/github/dedis/student20_pop/OrganizerActivity.java:79]: For the moment, in ordre to keep it simple, I assumed that it was thes cas. There is a comment about that suject In the specifications, it is mentionned the following : 
" The paradigm might be that you’re always anonymous if you’re an ordinary attendee and always non-anonymous if you’re an organizer or witness. "
But I think we can still change that in the future

File: app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java (hunk 3 above)
Comment (AlexisTabin) [reply app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java:52]: For the moment, we do not use theses fields at all

File: app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java (hunk 2 above)
Comment (AlexisTabin) [reply app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java]: Removed

**Comment by jbmichel14** [comment]
OK Good job!

**Review by jbmichel14 — APPROVED** [review]

**Review by GabrielFleischer — APPROVED** [review]
Everything seems fine to me

**Pull request merged**